cmake/configure_linux.cmake

```diff
@@ -0,0 +1,2 @@
+set(INLINE_POST     ON CACHE BOOL "Enable inline post" FORCE)
```

Comment:
We should definitely maintain separate build targets linux.gnu and linux.intel, even if the cmake configure file looks the same for now. The module files are also called `modulefiles/linux.COMPILER/fv3`.

Reply:
which generic linux box loads modules?

Reply:
Fine.
I will add 2 files with one line each, in addition to `configure_linux.cmake`

Reply:
Also, that is not a modulefile, it is a shell script.

Reply:
fixed in 8b6879a

Reply:
I thought you are on holidays! We don't need the configure file w/o the compiler in the name, just the configure_linux.COMPILER.cmake. The way the build system works is that you specify a full identifier, which (except for the wcoss platforms for historical reasons) is MACHINE.COMPILER. cmake will the load the config file (the shell scripts) and afterwards look for the modulefile - that's why the two have to have the same full identifier in the name:
```
cmake/configure_foo.bar.cmake
modulefiles/foo.bar/fv3
```
The module file for generic platforms is itself just a shell script (and the logic in `compile.sh` takes care of sourcing it and not loading it as a module).